### DISCUSSION TITLE
How to make each SRT entry longer in length (IGNORE)

### DISCUSSION THREAD

**Discussion by rioharper** [openai/whisper#671] [Q&A] [answered]
Hello! I am trying to do some voice processing and I find the length of each "entry" or whatever its called a little short. I found a previous discussion saying if I change timestamp_logprob in decoding.py to something like "timestamp_logprob > 4" on line 440. However, the "apply" def seems to not even be called during transcription, so Im a little lost

Example: 

"The man stepped in the hallway"
"hearing a noise behind him" 
->
"The man stepped in the hallway hearing a noise behind him"

Any help would be appreciated!

**Comment by jongwook** [comment, marked as answer]
Merging short segments could be done as a post-processing step (which won't have to involve deep learning or PyTorch), where you'd just merge short segments into one. Libraries like [`srt`](https://pypi.org/project/srt/) or [`pysrt`](https://pypi.org/project/pysrt/) could be handy.
